<!-- -*- mode: XML -*- -->

<launch>
  <arg name="rviz" default="true" />

  <arg name="config_path" default = "$(find monolio)/config/kitti_config_08.yaml" />
  <node pkg = "monolio" type = "monolio_estimator_node" name ="monolio_estimator" output = "screen">
        <param name = "config_file" type= "string" value ="$(arg config_path)"/>
  </node>
  
 <!--<arg name="loop_config_path" default = "$(find monolio)/config/kitti_loop_config_00.yaml" />
  <node pkg = "monolio" type = "monolio_loop_detection_node" name ="monolio_loop_detection" output = "screen">
        <param name = "loop_config_file" type= "string" value ="$(arg loop_config_path)"/>
  </node> -->
   


</launch>
